distributed/dashboard/components/scheduler.py

```diff
@@ -829,58 +834,103 @@ def __init__(self, scheduler, **kwargs):
                     "time": [],
                     "read_bytes": [],
                     "write_bytes": [],
+                    "cpu": [],
+                    "memory": [],
                 }
             )
 
             update(self.source, self.get_data())
 
             x_range = DataRange1d(follow="end", follow_interval=20000, range_padding=0)
 
-            self.root = figure(
+            self.bandwidth = figure(
                 title="Workers Network Bandwidth Timeseries",
                 x_axis_type="datetime",
                 tools="",
                 x_range=x_range,
-                id="bk-worker-net-bandwidth-ts",
+                id="bk-worker-network-bandwidth-ts",
                 name="worker_network_bandwidth-timeseries",
                 **kwargs,
             )
 
-            self.root.line(
+            self.bandwidth.line(
                 source=self.source,
                 x="time",
                 y="read_bytes",
                 color="red",
                 legend_label="read (sum)",
             )
-            self.root.line(
+            self.bandwidth.line(
                 source=self.source,
                 x="time",
                 y="write_bytes",
                 color="blue",
                 legend_label="write (sum)",
             )
 
-        self.root.yaxis.axis_label = "bytes / second"
+            self.bandwidth.yaxis.axis_label = "bytes / second"
+            self.bandwidth.yaxis[0].formatter = NumeralTickFormatter(format="0.0b")
+            self.bandwidth.y_range.start = 0
+
+            self.cpu = figure(
+                title="Workers CPU Timeseries",
+                x_axis_type="datetime",
+                tools="",
+                x_range=x_range,
+                id="bk-worker-network-cpu-ts",
+                name="worker_network_cpu-timeseries",
```

Comment:
@jrbourbeau I'm not sure we use `name` for anything, but we usually keep it across plots. Should I remove the world "network" in the cpu/memory and disk entries for "id" and "name"

Reply:
Name is used when embedding Bokeh charts in HTML/CSS web pages